yongwook/sw_1206_view.java

```diff
@@ -0,0 +1,41 @@
+package practice;
+
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class sw_1206_view {
+	public static void main(String[] args) {
+		
+		Scanner sc = new Scanner(System.in);
+		
+		int tc = 10;
+		
+		for (int i = 0; i < tc; i++) {
+			int result = 0;
+			int N = sc.nextInt();
+			int high[] = new int[N];
+			for (int j = 0; j < N; j++) {
+				 high[j] = sc.nextInt();
+			}
+			
+			for (int j = 2; j < N-2; j++) {
+				
+				int status[] = new int[4];
+				
+				status[0] = high[j] -high[j-1] ;
+				status[1] = high[j] -high[j+1] ;
+				status[2] = high[j] -high[j-2] ;
+				status[3] = high[j] -high[j+2] ;
```

Comment:
int[] dir = { -2, -1, 1, 2 } 
선언해주고

```suggestion
				int status[] = new int[4];
				
				for(int k=0; k<4; k++){
				status[k] = high[j] -high[dir[k]] ;
				}
```

이런식으로 줄일 수 있을 것 같아요!